tools/Generator.CLI/Program.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+
+namespace Generator.CLI
+{
+	class Program
+	{
+		static int Main(string[] args)
+		{
+			var runner = new CommandDotNet.AppRunner<ConsoleShell>();
+#if (DEBUG)
+			runner.Run(args);
+			while (true)
```

Comment:
@jnm2 Is that a british "interesting" ([see])(https://www.telegraph.co.uk/news/newstopics/howaboutthat/10280244/Translation-table-explaining-the-truth-behind-British-politeness-becomes-internet-hit.html) or a rest of the world "interesting" ? I don't mind changing it. Still wondering nevertheless

Reply:
I'm not British, so thanks for making me aware of that! (I'm in Pennsylvania.)

Wow, that table is... interesting! 😁

Reply:
Ah. Only googled Lancaster because it sounded british and got the one in england. Always got to check which kind of words are used when talking to british people, because the semantics of politeness distort the communication :p Also why i am not a big fan of written sarcasm - anyway. Resolving this one too then :)

Reply:
There are 23 cities in the US named Lancaster. I'm in the one that was capitol of the US for a single day. :D